app/javascript/packs/application.js

```diff
@@ -18,3 +18,8 @@ global.mapsLoadedCallback = function() {
     instance.initMap() ;
   }
 }
+
+// custom Google Analytics events
+import { descriptionSummary, descriptionToggleEvent } from 'analytics/school_description_helper.js';
+
+descriptionSummary.addEventListener('toggle', descriptionToggleEvent);
```

Comment:
My reading of this change is that since the last review it's switched from registering the event listener on a single page which was including the custom page, to instead registering the event listener on all pages since its part of the main pack?

My suggestion is to add a function within the pack which will register the event listener, and then add a js block where you were previously doing a js include, which calls that function? Then it will still only affect a single page.

Reply:
Sorry, so you suggest just registering the listener from within a `<script>` tag on the page in question?